zookeeper/data.py

```diff
@@ -114,7 +117,12 @@ def test_data(self, batch_size):
     def _get_eval_data(self, dataset, batch_size):
         return (
             dataset.repeat()
-            .map(self.map_fn, num_parallel_calls=tf.data.experimental.AUTOTUNE)
+            .map(
+                self.map_fn,
+                num_parallel_calls=tf.data.experimental.AUTOTUNE
+                if self.autotune
+                else 1 * batch_size,
+            )
             .batch(batch_size)
-            .prefetch(tf.data.experimental.AUTOTUNE)
+            .prefetch(tf.data.experimental.AUTOTUNE if self.autotune else 2)
```

Comment:
Would we be able to set the prefetch amount to be configurable? On a side note I think before it was around 8

Reply:
@lgeiger how do you feel about this?